### DISCUSSION TITLE
Copy and rename files after build

### DISCUSSION THREAD

**Discussion by bdarbonneau** [facebook/docusaurus#8527] [Q&A]
Each time I run 'npm build' I would like to also :
- copy the content of the build folder to C:\myfolder
- rename files in C:\myfolder\  from *.html to *.aspx

How can this be added to the build actions ?

**Comment by nicocodemance** [comment]
If you are using ubuntu:
```js
{
   "build": "rename 's/.html/.aspx' C:/myfolder/**/*.html && [BUILD COMMAND]"
}
```

Or if you are using windows
```js
{
    "build": "ren C:/myfolder/**/*.html *.aspx && [BUILD COMMAND]"
}
```

**Reply by slorber** [reply]
It doesn't need to be added to the "docusaurus build" action, you can plug your own logic from the outside by simply chaining shell commands, or chaining commands in the npm "build" script (that defaults to `docusaurus build` but could do more as suggested by @nicholidev )


For example, you can just run multiple commands in a row

```sh
npm run build && echo "do whatever you want here"
```
